### PR TITLE
Fix: Select & Cascader didn't show corresponding style when changing size property

### LINKED ISSUE #16725: [Bug Report] drop-down box class bugs about size
### Element UI version
2.10.1

### OS/Browsers version
all

### Vue version
2.6.10

### Reproduction Link
https://codepen.io/pen/?&editable=true

### Steps to reproduce
1. 打开网址 https://codepen.io/pen/?&editable=true
2. 在el-select 标签写入size="mini"。如代码： 
```html
<el-select v-model="value" placeholder="请选择" size="mini">
```
3. 可以看到样式 .el-scrollbar__view.el-select-dropdown__list 的下拉框（就是 el-option) 中字体(font-size)并没有跟随 size 变化。

### What is Expected?
el-option 中字体(font-size)跟随 size 变化

### What is actually happening?
el-option 中字体(font-size)并没有跟随 size 变化

### PR DESCRIPTION
**Pull request by nullptru**
Please make sure these boxes are checked before submitting your PR, thank you!

Close #16725

Add size property and corresponding css style to select-dropdown && cascader panel.

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by ziyoung** [comment]
@luckyCao 还包含一个 cascader 的改动，有时间看一下。

File: packages/theme-chalk/src/cascader-panel.scss [hunk 1]
```diff
@@ -13,6 +13,77 @@
     border: $--cascader-menu-border;
     border-radius: $--cascader-menu-radius;
   }
+  @include m(medium) {
```
Comment (ziyoung) [inline packages/theme-chalk/src/cascader-panel.scss]: var.scss 中提前定义了一些 font-size。可以直接使用的。
```scss
$--font-size-extra-large: 20px !default;
$--font-size-large: 18px !default;
$--font-size-medium: 16px !default;
$--font-size-base: 14px !default;

$--font-line-height-primary: 24px !default;
$--font-line-height-secondary: 16px !default;
```

**Comment by ziyoung** [comment]
@nullptru  用 CSS 变量吧，其他修改看起来没问题。

File: packages/theme-chalk/src/cascader-panel.scss (hunk 1 above)
Comment (nullptru) [reply packages/theme-chalk/src/cascader-panel.scss]: OK，一会修改下.

**Comment by nullptru** [comment]
@ziyoung font-size 用变量替换完了...

**Pull request closed without merging**